android/app/src/main/java/network/mysterium/wallet/WalletTopupAmountFragment.kt

```diff
@@ -0,0 +1,48 @@
+package network.mysterium.wallet
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import network.mysterium.MainApplication
+import network.mysterium.ui.onChange
+import network.mysterium.vpn.databinding.FragmentWalletTopupAmountBinding
+import java.math.BigDecimal
+
+class WalletTopupAmountFragment : Fragment() {
+
+    private lateinit var viewModel: WalletTopupViewModel
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val binding = FragmentWalletTopupAmountBinding.inflate(inflater, container, false)
+        viewModel = (requireActivity().application as MainApplication).appContainer.walletTopupViewModel
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this.activity
+
+        binding.toolbar.setNavigationOnClickListener {
+            requireActivity().onBackPressed()
+        }
+        binding.amount20.setOnClickListener {
+            viewModel.topupAmount.value = BigDecimal(20)
```

Comment:
Could we have these values somewhere in config, settings or fetched from some service (e.g. pilvytis)?

The problem is that MYST is volatile asset and 20MYST in one month could mean much more than now...

Reply:
That's a good idea. I've created tickets for this:
https://github.com/mysteriumnetwork/node/issues/2916 (API) 
https://github.com/mysteriumnetwork/node/issues/2917 (Android)
https://github.com/mysteriumnetwork/node/issues/2915 (Desktop)